pages/discover.js

```diff
@@ -20,62 +20,61 @@ export default function Discover() {
-        <div className="max-w-md basis-120">
+        <div className="max-w-lg basis-120">
           <h3 className="text-xl sm:text-2xl font-bold text-transparent bg-clip-text bg-gradient-to-r from-blue-500 to-sky-500">
             A lightweight and open-source code base.
           </h3>
           <div className="pt-4 text-base">
-            You can use Condensation for commercial purpose for free, it is organised in lightweight
-            modules which can be audited by a single person. Moreover, it can be customized with new
-            implementations for specific purposes such as preferred encryption methods or
-            pre-defined conflict resolution rules.
+            Condensation can be used for commercial purpose without any fees. It is organized in
+            three lightweight modules that can be easily audited. Moreover, its implementation can
+            be customized for specific needs, such as preferred encryption algorithms or pre-defined
+            conflict resolution rules.
           </div>
         </div>
 
-        <div className="max-w-md basis-120">
+        <div className="max-w-lg basis-120">
           <h3 className="text-xl sm:text-2xl font-bold text-transparent bg-clip-text bg-gradient-to-r from-blue-500 to-sky-500">
-            Generic to build tailored solutions.
+            Generic enough to build tailored solutions.
           </h3>
           <div className="pt-4">
-            You can manage the complete lifecycle of data with Condensation, that’s why it is called
-            a data-system. It excels with synchronization performances in a fully distributed setup,
-            provides a single protocol to connect multiple devices with end-to-end data security,
-            and stores data without centralization constraints.
+            Condensation is a data-system, i.e able to manage a complete data lifecycle. It excels
+            particularly at data synchronization in a distributed set-up. Its main advantages are a
+            single protocol to connect multiple devices, constitutional data security from end to
+            end, and data stores free from centralization constraints.
           </div>
         </div>
 
-        <div className="max-w-md basis-120">
+        <div className="max-w-lg basis-120">
           <h3 className="text-xl sm:text-2xl font-bold text-transparent bg-clip-text bg-gradient-to-r from-blue-500 to-sky-500">
             Secured with market proven algorithms.
           </h3>
           <div className="pt-4">
-            The implementation is independent from dependencies and use basic cryptographic
-            primitives extracted from challenged open-source projects. Moreover, the choice of the
-            algorithms fits to nowadays device capabilities to ensure run time performances that
-            fits to connected objects, or high speed execution of apps.
+            Condensation implementation is self-reliant from software dependencies. It uses
```

Comment:
- from dependencies to other libraries 
- I am not a big fan of self-reliant, its a bit hard to understand for the common english speaker like me :p